src/assets/BulkSessionCreationNew.csv

```diff
@@ -1,5 +1,6 @@
-Action,id,title,description,type,Mentor(Email/Mobile Num),Mentees(Email/Mobile Num),Date(DD-MM-YYYY),Time Zone(IST/UTC),Time (24 hrs),Duration(Min),recommended_for,categories,medium,Meeting Platform,Meeting Link or Meeting ID,Meeting Passcode (if needed)
+Action,id,title,description,type,Mentor(Email),Mentees(Email),Date(DD-MM-YYYY),Time Zone(IST/UTC),Time (24 hrs),Duration(Min),recommended_for,categories,medium,Meeting Platform,Meeting Link,Meeting Passcode (if needed)

```

Comment:
@sumanvpacewisdom why mobile Num is removed

Reply:
@rakeshSgr Products have changed the csv and asked to change , we are using only email for now is what they said